articles/quickstart/webapp/nodejs-ea/01-login.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login, logout, and profile to a Node.js Express application.
+budicon: 448
+topics:
+  - quickstarts
+  - webapp
+  - login
+  - user profile
+  - logout
+  - nodejs
+contentType: tutorial
+useCase: quickstart
+github:
+  path: 01-Login
+---
+
+<!-- markdownlint-disable MD002 -->
+## Getting Started
+In this quickstart you will learn how to implement authentication in an Express application using the OIDC Compliant library [Express OIDC](https://github.com/auth0/express-openid-connect) authored and maintained by Auth0.
+
+::: warning
+Express OIDC is still in an [early access stage](/).
+:::
+
+### Install Dependencies
+
+```sh
+# Using NPM
+npm install express-openid-connect express-session
+
+# Using Yarn
+yarn add express-openid-connect express-session
+```
+
+## Configure Auth0
```

Comment:
I believe these should use includes like the [Passport one](https://raw.githubusercontent.com/auth0/docs/master/articles/quickstart/webapp/nodejs/01-login.md)

Reply:
I was trying to avoid all the boilerplate that comes loaded in those partials.  But addressing the length/brevity of that content could be addressed separately.

Reply:
Fair enough. As long as it was intentional 👍